code/web/src/modules/user/EditProfile.js

```diff
@@ -156,26 +144,20 @@ const EditProfile = (props) => {
 
                         onChange={e => updateDescription(e.target.value)}
                     />
-                </GridCell>
-
                 {/* Upload File */}
-                <div style={{ marginTop: '1em' }}>
+                <section style={{ margin: '5%', display: 'block' }}>
+                    <label htmlFor="profile-picture" style={{margin: '0 5%'}}>Update Profile Picture:</label>
                     <input
                       type="file"
+                      name="profile-picture"
                       onChange={e => onUpload(e)}
                     />
-                  </div>
-
-                  {/* Uploaded image */}
-                  {/* {renderIf(this.state.product.image !== '', () => (
-                    <img src={routeImage + this.state.product.image} alt="Product Image"
-                         style={{ width: 200, marginTop: '1em' }}/>
-                  {/* ))} */}
-
+                  </section>
+                </GridCell>
 
-                <GridCell>
+                <GridCell style={{ margin: '0 5%'}} >
                     <Link to={userRoutes.profile.path}>
-                      <Button theme="primary" onClick={handleClick}>Save Changes</Button>
+                      <Button theme="primary" onClick={handleClick} style={{display: 'block', marginBottom: '5%'}}>Save Changes</Button>
                     </Link>
                     <Link to={userRoutes.profile.path}>
                         <Button onClick={clearInputs}theme="secondary">Abort</Button>
```

Comment:
Assuming this looks good, weird bug where the profile page might look different from machine to machine or branch from branch